RecoEgamma/PhotonIdentification/data/Fall17/effAreaPhotons_cone03_pfNeutralHadrons_90percentBased_TrueVtx.txt

```diff
@@ -0,0 +1,16 @@
+# This file contains Effective Area constants for 
+# computing pile-up corrections for the neutral hadron isolation
+# for a photon object.
+#
+#  The effective areas are derived based on the 90% contour method.
+# These effective areas are also found on the following slides:
+#      https://indico.cern.ch/event/491517/contributions/2349134/attachments/1359450/2056689/CutBasedPhotonID_24-10-2016.p
+#
+# |eta| min   |eta| max   effective area
+0.0000         1.0000        0.064
+1.0000         1.4790        0.110
+1.4790         2.0000        0.076
+2.0000         2.2000        0.023
+2.2000         2.3000        0.151
+2.3000         2.4000        0.0001
```

Comment:
I was expecting moderately similar values
- between this 90percentBased_TrueVtx.txt and 90percentBased.txt file, but they are rather different
- between 2.2-2.3 , 2.3-2.4 and 2.0-2.2, but they are very different especially here. Please confirm that correct values were transferred to these text files.

Reply:
Typo in 0.151 number fixed here:
https://github.com/lsoffi/cmssw/blob/CMSSW_9_4_0_pre3_TnP/RecoEgamma/PhotonIdentification/data/Fall17/effAreaPhotons_cone03_pfNeutralHadrons_90percentBased_TrueVtx.txt

Reply:
what about 0.0001 ? this looks suspiciously different from the neighboring 0.0151 and 0.013.